src/rules/effects/prefer-concat-latest-from.ts

```diff
@@ -25,7 +25,6 @@ export default ESLintUtils.RuleCreator(docsUrl)<Options, MessageIds>({
       recommended: 'warn',
       suggestion: true,
     },
-    fixable: 'code',
```

Comment:
We should add this soon as we've upgrade this rule to provide a `fix`.

Reply:
What do you mean?